Mesh_3/examples/Mesh_3/mesh_3D_weighted_image.cpp

```diff
@@ -0,0 +1,71 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Mesh_triangulation_3.h>
+#include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#include <CGAL/Mesh_criteria_3.h>
+
+#include <CGAL/Labeled_mesh_domain_3.h>
+#include <CGAL/Mesh_3/generate_label_weights.h>
+#include <CGAL/make_mesh_3.h>
+#include <CGAL/Image_3.h>
+#include <CGAL/IO/File_binary_mesh_3.h>
+
+// Domain
+using K = CGAL::Exact_predicates_inexact_constructions_kernel;
+using Mesh_domain = CGAL::Labeled_mesh_domain_3<K>;
+
+using Concurrency_tag =
+#ifdef CGAL_CONCURRENT_MESH_3
+  CGAL::Parallel_tag;
+#else
+  CGAL::Sequential_tag;
+#endif
+
+// Triangulation
+using Tr   = CGAL::Mesh_triangulation_3<Mesh_domain, CGAL::Default, Concurrency_tag>::type;
+using C3t3 = CGAL::Mesh_complex_3_in_triangulation_3<Tr>;
+
+// Criteria
+using Mesh_criteria = CGAL::Mesh_criteria_3<Tr>;
+
+// To avoid verbose function and named parameters call
+using namespace CGAL::parameters;
+
+int main(int argc, char* argv[])
+{
+  /// [Loads image]
+  const char* fname = (argc > 1) ? argv[1] : "data/liver.inr.gz";
+  CGAL::Image_3 image;
+  if(!image.read(fname)){
+    std::cerr << "Error: Cannot read file " <<  fname << std::endl;
+    return EXIT_FAILURE;
+  }
+  /// [Loads image]
+
+  /// [Domain creation]
+  const float sigma = 10.f;
```

Comment:
What are the results if `sigma` is a lot smaller, like `1.f` or `2.f`?

Reply:
It's not very different on liver. Actually this value is absolute and should depend on the size of the bbox of the input

Reply:
That parameter sigma is without dimension, and it only acts on the blurring algorithm, to blur the indicator images.